guides/source/active_job_basics.md

```diff
@@ -131,77 +119,284 @@ That's it!
+
+For recurring tasks, any errors encountered while enqueuing are logged, but they won’t bubble up. Read more about [Errors When Enqueuing in the Solid Queue documentation](https://github.com/rails/solid_queue?tab=readme-ov-file#errors-when-enqueuing).
+
+### Concurrency Controls
+
+Solid Queue extends Active Job with concurrency controls, allowing you to limit how many jobs of a certain type or with specific arguments can run at the same time. If a job exceeds the limit, it will be blocked until another job finishes or the duration expires. For example:
+
+```ruby
+class MyJob < ApplicationJob
+  limits_concurrency to: 2, key: ->(contact) { contact.account }, duration: 5.minutes
+
+  def perform(contact)
+    # perform job logic
   end
 end
 ```
 
-You can also configure your backend on a per job basis:
+In this example, only two `MyJob` instances for the same account will run concurrently. After that, other jobs will be blocked until one completes.
+
+The `group` parameter can be used to control concurrency across different job types. For instance, two different job classes that use the same group will have their concurrency limited together:
 
 ```ruby
-class GuestsCleanupJob < ApplicationJob
-  self.queue_adapter = :resque
-  # ...
+class Box::MovePostingsByContactToDesignatedBoxJob < ApplicationJob
+  limits_concurrency key: ->(contact) { contact }, duration: 15.minutes, group: "ContactActions"
 end
 
-# Now your job will use `resque` as its backend queue adapter, overriding what
-# was configured in `config.active_job.queue_adapter`.
+class Bundle::RebundlePostingsJob < ApplicationJob
+  limits_concurrency key: ->(bundle) { bundle.contact }, duration: 15.minutes, group: "ContactActions"
+end
 ```
 
-[`config.active_job.queue_adapter`]: configuring.html#config-active-job-queue-adapter
+This ensures that only one job for a given contact can run at a time, regardless of the job class.
 
-### Starting the Backend
+Read more about [Concurrency Controls in the Solid Queue documentation](https://github.com/rails/solid_queue?tab=readme-ov-file#concurrency-controls).
 
-Since jobs run in parallel to your Rails application, most queuing libraries
-require that you start a library-specific queuing service (in addition to
-starting your Rails app) for the job processing to work. Refer to library
-documentation for instructions on starting your queue backend.
+### Error Reporting on Jobs
 
-Here is a noncomprehensive list of documentation:
+If your error tracking service doesn’t automatically report job errors, you can manually hook into Active Job to report exceptions. For example, you can add a `rescue_from` block in `ApplicationJob`:
```

Comment:
OPTIONAL: in the error guide we tried to keep the terms the same throughout so as to limit confusion. So, errors rather than exceptions (or even just 'them' in this case):

```suggestion
If your error tracking service doesn’t automatically report job errors, you can manually hook into Active Job to report them. For example, you can add a `rescue_from` block in `ApplicationJob`:
```